lib/FetchEnvs.js

```diff
@@ -288,7 +288,10 @@ function typeCast(value, type) {
       return (value.toLowerCase() === 'true');
     }
     case 'json': {
-      return JSON.parse(value);
+      if (value) {
```

Comment:
I see on line 280, the code checks for `null` and returns `undefined`. Are we getting to line 292 with a string that isn't valid JSON?

Reply:
Yes, if the value is specifically `""` the code throws an error, which this is guarding against.

Reply:
Examining the sample data, I see `""` which answers my question. I suggest we would have the same failure if the string were `{'some":"value"`. I suggest we wrap the parse with a try/catch to account for other similar parsing errors. The question then how the code communicates this failure out to the user by the return value, error logging, or some combination of these and others.

Reply:
I made some changes in [080e9c3](https://github.com/razee-io/razeedeploy-core/pull/311/commits/080e9c3c37641b7a55334f5602ec75c91c116345). In this case I'm just throwing a more helpful error that includes the ref name and value that couldn't be parsed. An end user will be able to observe this error message on their MTP and should aid in debugging.